metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bigcommerce-monit",
+  "version": "1.0.0",
+  "author": "Bigcommerce - Technical Operations <techops@bigcommerce.com>",
+  "license": "Proprietary",
```

Comment:
You should probably match all of this to the module we're forked from: https://github.com/farstarinc/puppet-module-monit